src/components/Votes.jsx

```diff
@@ -1,4 +1,34 @@
-function Votes({votes}){
-    return <span><button>-</button>Votes: {votes} <button>+</button></span>
+import {useState} from "react";
+
+
+function Votes({votes, articleId}){
+    const [currentVote, setCurrentVote] = useState(votes)
+    const [disabled, setDisabled] = useState(false)
+
+    async function changeVotes(e){
+        try {
+        const incOrDec = e.target.innerText === "+" ? 1 : -1
+
+            setCurrentVote(currentVote + incOrDec)
+            const articleCopy = {article_id:articleId,
+                inc_votes: incOrDec
+            }
+            const res = await fetch(`https://be-northcoder-news.onrender.com/api/articles/${articleId}`, {
```

Comment:
Please extract this request into a utils file using axios and a base URL instead of 'fetch'.